### PR TITLE
Bibox v4 fetch ticker

### PR DESCRIPTION
**Pull request by samgermain**
```
% bibox fetchTicker LTC/BTC                                                  % bibox fetchTicker BTC/USDT
2022-10-19T02:14:36.868Z                                                     2022-10-19T02:17:10.644Z
Node.js: v18.4.0                                                             Node.js: v18.4.0
CCXT v2.0.38                                                                 CCXT v2.0.38
bibox.fetchTicker (LTC/BTC)                                                  bibox.fetchTicker (BTC/USDT)
2022-10-19T02:14:39.119Z iteration 0 passed in 779 ms                        2022-10-19T02:17:12.197Z iteration 0 passed in 362 ms

{                                                                            {
  symbol: 'LTC/BTC',                                                           symbol: 'BTC/USDT',
  timestamp: 1666145678000,                                                    timestamp: 1666145832000,
  datetime: '2022-10-19T02:14:38.000Z',                                        datetime: '2022-10-19T02:17:12.000Z',
  high: 0.00269338,                                                            high: 19699.9,
  low: 0.00262959,                                                             low: 19112.3,
  bid: 0.0026787,                                                              bid: 19304.1,
  bidVolume: 5.9805,                                                           bidVolume: 0.373902,
  ask: 0.00268594,                                                             ask: 19304.7,
  askVolume: 16.0913,                                                          askVolume: 0.429188,
  vwap: 376.781166898081,                                                      vwap: 0.000051365526480858,
  open: 0.00263551,                                                            open: 19527,
  close: 0.00267885,                                                           close: 19304.1,
  last: 0.00267885,                                                            last: 19304.1,
  previousClose: undefined,                                                    previousClose: undefined,
  change: 0.0134,                                                              change: -0.0116,
  percentage: 508.4404915936574,                                               percentage: -0.000059404926512,
  average: 0.00265718,                                                         average: 19415.55,
  baseVolume: 13.45415282227,                                                  baseVolume: 12471598.8696956,
  quoteVolume: 5069.2714,                                                      quoteVolume: 640.610242,
  info: {                                                                      info: {
    s: 'LTC_BTC',                                                                s: 'BTC_USDT',
    t: '1666145678000',                                                          t: '1666145832000',
    o: '0.00263551',                                                             o: '19527',
    h: '0.00269338',                                                             h: '19699.9',
    l: '0.00262959',                                                             l: '19112.3',
    p: '0.00267885',                                                             p: '19304.1',
    q: '0.097',                                                                  q: '0.002341',
    v: '5069.2714',                                                              v: '640.610242',
    a: '13.45415282227',                                                         a: '12471598.8696956',
    c: '0.0134',                                                                 c: '-0.0116',
    n: '40291',                                                                  n: '158712',
    f: '15897305',                                                               f: '58233124',
    bp: '0.0026787',                                                             bp: '19304.1',
    bq: '5.9805',                                                                bq: '0.373902',
    ap: '0.00268594',                                                            ap: '19304.7',
    aq: '16.0913'                                                                aq: '0.429188'
  }                                                                            }
}                                                                            }
2022-10-19T02:14:39.119Z iteration 1 passed in 779 ms                        2022-10-19T02:17:12.197Z iteration 1 passed in 362 ms
```

```
2022-10-19T02:19:33.478Z
Node.js: v18.4.0
CCXT v2.0.38
bibox.fetchTickers ()
2022-10-19T02:19:35.995Z iteration 0 passed in 1088 ms

{
  'BIX/BTC': {
    symbol: 'BIX/BTC',
    timestamp: undefined,
    datetime: undefined,
    high: 8e-7,
    low: 7.9e-7,
    bid: undefined,
    bidVolume: undefined,
    ask: undefined,
    askVolume: undefined,
    vwap: 7.98015899345e-7,
    open: 7.9e-7,
    close: 8e-7,
    last: 8e-7,
    previousClose: undefined,
    change: 1e-8,
    percentage: 1.27,
    average: undefined,
    baseVolume: 6691596,
    quoteVolume: 5.34,
    info: {
      is_hide: '0',
      high_cny: '0.1092',
      amount: '5.34',
      coin_symbol: 'BIX',
      last: '0.00000080',
      currency_symbol: 'BTC',
      change: '+0.00000001',
      low_cny: '0.1078',
      base_last_cny: '0.10918434',
      area_id: '7',
      percent: '+1.27%',
      last_cny: '0.1092',
      high: '0.00000080',
      low: '0.00000079',
      pair_type: '0',
      last_usd: '0.0154',
      vol24H: '6691596',
      id: '1',
      high_usd: '0.0154',
      low_usd: '0.0153'
    }
  },
  ...
}
2022-10-19T02:19:35.995Z iteration 1 passed in 1088 ms
```

### CODE REVIEW

File: js/bibox.js [hunk 1]
```diff
@@ -429,17 +429,62 @@ module.exports = class bibox extends Exchange {
...
+        //        low: '0.00000079',
+        //        pair_type: '0',
+        //        last_usd: '0.0155',
+        //        vol24H: '6697325',
+        //        id: '1',
+        //        high_usd: '0.0155',
+        //        low_usd: '0.0153'
+        //    }
+        //
+        const timestamp = this.safeInteger2 (ticker, 'timestamp', 't');
```
Comment (carlosmiei) [inline js/bibox.js:479]: @samgermain change/percentage seems a bit off 
image

Comment (samgermain) [reply js/bibox.js:479]: That's the value that the exchange is giving me, it's not very far off though, `1 - (19526.3 / 19265.6) = -0.01353189104` it's a ratio though

Comment (carlosmiei) [reply js/bibox.js:479]: @samgermain yeah but `change` should get the absolute value, and this differs from the v1 value

**Comment by carlosmiei** [comment]
LGTM ✅ 

```
p bibox fetchTicker "BTC/USDT" 
Python v3.10.6
CCXT v2.0.38
bibox.fetchTicker(BTC/USDT)
{'ask': 19181.7,
 'askVolume': 0.402254,
 'average': 19376.95,
 'baseVolume': 10522921.3142568,
 'bid': 19181.5,
 'bidVolume': 0.164977,
 'change': -390.9,
 'close': 19181.5,
 'datetime': '2022-10-19T11:29:58.000Z',
 'high': 19691.2,
 'info': {'a': '10522921.3142568',
          'ap': '19181.7',
          'aq': '0.402254',
          'bp': '19181.5',
          'bq': '0.164977',
          'c': '-0.02',
          'f': '58291338',
          'h': '19691.2',
          'l': '19112.3',
          'n': '157872',
          'o': '19572.4',
          'p': '19181.5',
          'q': '0.001003',
          's': 'BTC_USDT',
          't': '1666178998000',
          'v': '544.598717'},
 'last': 19181.5,
 'low': 19112.3,
 'open': 19572.4,
 'percentage': -1.9972001389712044,
 'previousClose': None,
 'quoteVolume': 544.598717,
 'symbol': 'BTC/USDT',
 'timestamp': 1666178998000,
 'vwap': 5.1753567354168e-05}
 ```
 ```
 p bibox fetchTickers '["BTC/USDT"]'
Python v3.10.6
CCXT v2.0.38
bibox.fetchTickers(['BTC/USDT'])
{'BTC/USDT': {'ask': None,
              'askVolume': None,
              'average': None,
              'baseVolume': 543.0,
              'bid': None,
              'bidVolume': None,
              'change': -397.4,
              'close': 19180.5,
              'datetime': None,
              'high': 19691.2,
              'info': {'amount': '10508754.80',
                       'area_id': '15',
                       'base_last_cny': '135605.86244510',
                       'change': '-397.4000',
                       'coin_symbol': 'BTC',
                       'currency_symbol': 'USDT',
                       'high': '19691.2000',
                       'high_cny': '139216.5042',
                       'high_usd': '19691.2000',
                       'id': '36',
                       'is_hide': '0',
                       'last': '19180.5000',
                       'last_cny': '135605.8624',
                       'last_usd': '19180.5000',
                       'low': '19112.3000',
                       'low_cny': '135123.6894',
                       'low_usd': '19112.3000',
                       'pair_type': '0',
                       'percent': '-2.02%',
                       'vol24H': '543'},
              'last': 19180.5,
              'low': 19112.3,
              'open': 19577.9,
              'percentage': -2.02,
              'previousClose': None,
              'quoteVolume': 10508754.8,
              'symbol': 'BTC/USDT',
              'timestamp': None,
              'vwap': 19353.139594843462}}
```

**Pull request merged**